### PR TITLE
In-Person Payments: represent transactions as card payments.

### LINKED ISSUE #4101: In-Person Payments: ensure the proper logo is displayed in listings for IPP/Interac
### Description

We need to clarify and ensure which logo should be displayed when paying with Visa/Master card (incl. Interac).
Context: I couldn't find any related discussions and would like to re-evaluate the status here.

### PR DESCRIPTION
**Pull request by kalessil**
Fixes https://github.com/Automattic/woocommerce-payments/issues/4101

#### Changes proposed in this Pull Request

- Fixes payment method display for In-Person Payments

#### Testing instructions

* See the server PR available via ticket
* Make In-Person payment (regular or Interac)
* Ensure client lists the payment as card apyment

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

File: client/utils/account-fees.tsx [hunk 1]
```diff
@@ -315,7 +315,8 @@ export const getTransactionsPaymentMethodName = (
 		case 'card':
 			return __( 'Card transactions', 'woocommerce-payments' );
 		case 'card_present':
-			return __( 'In-person transactions', 'woocommerce-payments' );
+		case 'interac_present':
+			return __( 'In-Person Payment', 'woocommerce-payments' );
```
Comment (kalessil) [inline client/utils/account-fees.tsx:319]: `In-Person Payments`

Comment (kalessil) [inline client/utils/account-fees.tsx:319]: `In-Person Payments`

**Review by mgascam — APPROVED** [review]
LGTM!
✅ Card present payments display card's brand in transactions

**Review by dpaun1985 — APPROVED** [review]
LGTM

**Comment by kalessil** [comment]
On-hold until feedback in pdjVC1-yr-p2

**Pull request closed without merging**